content/spin/v3/triggers.md

```diff
@@ -156,109 +155,21 @@ workdir = "rust-redis-trigger-example"
 watch = ["src/**/*.rs", "Cargo.toml"]                             
 ```
 
-## Cron Trigger
-
-Spin has experimental support for creating and running components on a schedule. Please note that there are only working Cron Trigger app samples written in [Rust](https://github.com/fermyon/spin-trigger-cron/tree/main/guest-rust) and [Python](https://github.com/fermyon/spin-trigger-cron/tree/main/guest-python) at present.
-
-> Please note: You can not `spin deploy` an application to Fermyon Cloud if it uses `cron` because non-HTTP triggers are not supported in Fermyon Cloud.
-
-Let's look at how the [experimental Cron trigger for Spin](https://github.com/fermyon/spin-trigger-cron) allows you to deploy an application that runs on a schedule. A Cron trigger maps a cron expression (a schedule) to a specific component. For example:
-
-<!-- @nocpy -->
-
-```toml
-[[trigger.cron]]
-component = "hello-cron"
-cron_expression = "1/2 * * * * *"
-```
-
-> Note: The 7th field (year) for the `cron_expression` is optional.
-
-### Cron Trigger Expressions
-
-The expression is based on the crontab (cron table) syntax whereby each line is made up of 7 fields that represent the time to execute.
-
-```bash
-#  ┌──────────── sec (0–59)
-#  |    ┌───────────── min (0–59)
-#  |    │  ┌───────────── hour (0–23)
-#  |    │  │  ┌───────────── day of month (1–31)
-#  |    │  │  │  ┌───────────── month (1–12)
-#  |    │  │  │  │  ┌───────────── day of week (0–6)
-#  |    │  │  │  │  |  ┌─────────────- year
-#  |    │  │  │  │  |  │
-#  |    │  │  │  │  |  │
-  1/30  *  *  *  *  *  * 
-```
+## Available Triggers for Spin
 
-> For more information about setting the schedule, please see the [Spin Cron Trigger repository](https://github.com/fermyon/spin-trigger-cron?tab=readme-ov-file#trigger-configuration). 
+You can leverage different triggers as part of your Spin apps to address common requirements and build real-world, distributed applications with Spin.
 
-Let's look at a time-based workload inside a Rust application.
-
-### Installing the Cron Trigger Plugin
-
-First, we install the plugin:
-
-```bash
-spin plugins install --url https://github.com/fermyon/spin-trigger-cron/releases/download/canary/trigger-cron.json
-```
+### Common Triggers
```

Comment:
To me, i see the categorization slightly differently, partially because i feel "common" is a little vague.
1. Triggers built into the spin CLI (http and redis)
2. Triggers maintained by Spin (command, cron, sqs, mqtt)
3. Community triggers (kinesis)

I bubbled MQTT into spin because it is a part of spinkube but i see how that could be seen as a community trigger too.